deploy-manage/deploy/cloud-enterprise/find-endpoint-url.md

```diff
@@ -33,7 +39,7 @@ To connect to your Elasticsearch cluster, you need to look up the the cluster En
 Currently, we support the following ways of connecting to an Elasticsearch cluster:
 
 RESTful API with JSON over HTTP and HTTPS
-:   Used by the `curl` command and most programming languages that aren’t Java. To interact with your cluster, use your Elasticsearch cluster endpoint information from the **Overview** page in the Cloud UI. Port 9200 is used for plain text, insecure HTTP connections while port 9243 is used for HTTPS. Using HTTPS is generally recommended, as it is more secure.
+:   Used by the `curl` command and most programming languages that aren’t Java. To interact with your cluster, use your Elasticsearch cluster endpoint information from the deployment overview page in the Cloud UI. Port 9200 is used for plain text, insecure HTTP connections, while port 9243 is used for HTTPS. Using HTTPS is generally recommended, as it is more secure.
 
     If this is your first time using Elasticsearch, you can try out some `curl` commands to become familiar with the basics. If you’re on an operating system like macOS or Linux, you probably already have the `curl` command installed. For example, to connect to your cluster from the command line over HTTPS with the `curl` command:
 
```

Comment:
can you either delete the ingest methods section (line 71) or give the title heading formatting?

Reply:
I've added a heading, but I wouldn't mind deleting it, whatever you prefer :)